apps/namadillo/src/atoms/integrations/services.ts

```diff
@@ -159,3 +156,32 @@ export const queryAndStoreRpc = async <T>(
     throw err;
   }
 };
+
+export const updateIbcTransactionStatus = async (
+  rpc: string,
+  tx: IbcTransferTransactionData,
+  changeTransaction: (
+    hash: string,
+    update: Partial<IbcTransferTransactionData>
+  ) => void
+): Promise<void> => {
+  const client = await StargateClient.connect(rpc);
+  const successQueries = await queryForAck(client, tx);
+  if (successQueries.length > 0 && tx.hash) {
+    changeTransaction(tx.hash, {
+      status: "success",
+      progressStatus: "complete",
+      resultTxHash: successQueries[0].hash,
```

Comment:
Do we need `resultTxHash` for something?

Reply:
My idea is to have a link somewhere to redirect the users to a explorer where they can inspect the success / error ack transaction. Still need to figure out where to place this link and how / where we should define a default explorer